### DISCUSSION TITLE
Grafana - InfluxDB2 dashboards for VOLUMES

### DISCUSSION THREAD

**Discussion by vitomatics** [NetApp/harvest#964] [General]
Does someone have a dashboard for Netapp Volumes???
I've installed all of these but only get aggregates.

harvest_dashboard_aggregate.json
harvest_dashboard_cluster.json
harvest_dashboard_headroom.json
harvest_dashboard_metadata.json
harvest_dashboard_network.json
harvest_dashboard_shelf.json
harvest_dashboard_snapmirror.json
harvest_dashboard_svm.json

**Comment by vitomatics** [comment]
All these existing dashboards look great but don't see one for per volume

**Reply by cgrinds** [reply]
hi @vitomatics you're right, we haven't ported the Prometheus volume dashboard to InfluxDB yet. 95% of Harvest customers use Prometheus so it gets dashboards first. Are you interested in trying to translate the Prometheus one and contribute it? I'll create an issue and we can track there.

**Reply by vitomatics** [reply]
I'm certainly gonna try.   I'm wondering if it would be more effective to try converting the existing influx2 svm dashboard to volumes then to convert the Prometheus dashboard.  Thoughts?

**Reply by cgrinds** [reply]
Maybe you can use both, the influx svm dashboard will show you how the influx queries are structured and the Prometheus volume dashboard will show you which panels you want to bring over.
